app/src/main/java/crux/bphc/cms/activities/TokenViewModel.kt

```diff
@@ -0,0 +1,223 @@
+package crux.bphc.cms.activities
```

Comment:
Move this viewmodel into a separate package, either under `crux.bphc.cms.activities` or within a separate `crux.bphc.cms.viewmodels`. We don't want a cluttered codebase.